definitions/npm/bitcoinjs-lib_v2.x.x/flow_>=v0.17.x/bitcoinjs-lib_v2.x.x.js

```diff
@@ -97,6 +97,8 @@ declare module 'ecurve' {
 }
 // ---------- copypasta end ---- 
 
+import {Point as ECPoint} from 'ecurve';
```

Comment:
Hmmm... I wrote the TODO :) since I wanted to explain that I copy-paste the definitions instead of importing them :)

Anyway, bitcoin.js ECPair constructor requires Point from ecurve as a parameter (see other changes in the commit; the current version in the master is wrong - I made a mistake when writing it). How to do that, if not like that? I cannot write `Q: ?ecurve.Point` or anything similar. `ecurve` definition is copy-pasted as a whole in the start of the definition.

Also, I think the "test" runs successfully, so it seems to me that it typechecks like it should...